subsys/bluetooth/controller/ll_sw/ull_peripheral_iso.c

```diff
@@ -251,26 +254,59 @@ void ull_peripheral_iso_start(struct ll_conn *acl, uint32_t ticks_at_expire)
 {
 	struct ll_conn_iso_group *cig;
 	struct ll_conn_iso_stream *cis;
+	uint32_t acl_to_cig_ref_point;
+	uint32_t cis_offs_to_cig_ref;
 	uint32_t ticker_status;
 	uint32_t ticks_interval;
+	uint16_t cis_handle;
 	uint8_t ticker_id;
 
+	cis_handle = acl->llcp_cis.cis_handle;
+
 	cig = ll_conn_iso_group_get_by_id(acl->llcp_cis.cig_id);
-	cis = ll_conn_iso_stream_get(acl->llcp_cis.cis_handle);
+	cis = ll_conn_iso_stream_get(cis_handle);
+
+	cis_offs_to_cig_ref = cig->sync_delay - cis->sync_delay;
+
+	for (int i = 0; i < cig->lll.num_cis; i++) {
+		uint16_t handle = cig->cis_handles[i];
+
+		/* Find valid CIS handle, but exclude incoming */
+		if ((handle != 0xFFFF) && (handle != cis_handle)) {
+			/* Ticker already started - just set the offset and
+			 * assign LLL handle (now valid).
+			 */
+			cis->lll.ticks_offset = HAL_TICKER_US_TO_TICKS(cis_offs_to_cig_ref);
+			cis->lll.handle = cis_handle;
+			return;
+		}
+	}
+
+	cis->lll.ticks_offset = HAL_TICKER_US_TO_TICKS(cis_offs_to_cig_ref);
+	cis->lll.handle = cis_handle;
 
 	ticker_id = TICKER_ID_CONN_ISO_BASE +
 		    ll_conn_iso_group_handle_get(cig);
 	ticks_interval = HAL_TICKER_US_TO_TICKS(cig->iso_interval *
 						CONN_INT_UNIT_US);
 
-	/* Start CIS peripheral CIG ticker. TODO: Handle multiple CISes - only
-	 * start ticker for first CIS.
+	/* Establish the CIG reference point by adjusting ACL-to-CIS offset
+	 * (cis->offset) by the difference between CIG- and CIS sync delays.
 	 */
+	acl_to_cig_ref_point = cis->offset - cis_offs_to_cig_ref;
+
+	/* Make sure we have time to service first subevent. TODO: Improve
+	 * by skipping <n> interval(s) and incrementing event_count.
+	 */
+	LL_ASSERT(acl_to_cig_ref_point >= EVENT_OVERHEAD_START_US);
```

Comment:
`EVENT_OVERHEAD_START_US` is the CPU overhead before a successful coarse ticker timeout can be setup for a radio event. This includes CPU time to prepare the LLL and coarse ticker minimum ticks offset, but excludes the radio startup/settling delay. 

No action required to my comment. I can tune when getting the feature functional on nRF SoCs.

Reply:
Yes, you're right. This again became a bit "ot-centric". Will see if I can figure out what is needed, otherwise go ahead and adjust for nRF when you see fit.